README.md

```diff
@@ -90,6 +90,18 @@ Persisting five articles having each three different authors, each with differen
 $article = ArticleFactory::make(5)->with('Authors[3].Address.City.Country', ['name' => 'Kenya'])->persist();
 ```
 
+#### On the command line:
+Factories can also conveniently populate your database in order to test your application on the browser.
+The following command will persist 5 articles, each with 3 irish authors, considering that the `ArticleFactory` class features
+a `withThreeIrishAuthors()` method:
+```$xslt
+bin/cake fixture_factories_persist Authors -n 5 -m withThreeIrishAuthors
+```
+The option `--dry-run` or `-d` will display the output without persisting.
+The option `-c` will persist in the connection provided (default is `test`).
+
+The `fixture_factories_persist` command is featured on CakePHP 4 only (open to contribution for CakePHP 3).
+
```

Comment:
We could also add another parameter to allow the use of a string in the with method. This should be useful and very simple to implement

```$xslt
bin/cake fixture_factories_persist Authors -n 5 -m withThreeIrishAuthors -w "Address.City.Country"
```

Which would result in calling : 

```
AuthorsFActory::make(5)->with('Address.City.Country')->withThreeIrishAuthors();
```

Reply:
I agree, this is a good additional feature. Will implement that.